Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/OnboardingGenerateKeyViewController.swift

```diff
@@ -43,7 +52,30 @@ class OnboardingGenerateKeyViewController: UITableViewController {
         trackEvent(.generateOwnerOnboarding)
     }
 
-    @objc private func didTapNextButton(_ sender: Any) {        
+    @objc private func didTapNextButton(_ sender: Any) {
+        let vc = EnterAddressNameViewController()
+        vc.actionTitle = "Save"
+        vc.descriptionText = "Choose a name for the owner key. The name is only stored locally and will not be shared with Gnosis or any third parties."
+        vc.screenTitle = "Enter Key Name"
+        vc.trackingEvent = .enterKeyName
+        vc.placeholder = "Enter name"
+        vc.address = privateKey.address
+        vc.completion = { [unowned self, vc] name in
+            guard PrivateKeyController.importKey(privateKey, name: name, isDrivedFromSeedPhrase: true) else { return }
```

Comment:
I'm a bit confused - shouldn't we go directly to the key details after entering key name?

From the task description:
```
When pressed,
will save the seed phrase securely and the private key
Updates tracking user properties and triggers tracking events
Closes the flow and opens the key details
Shows snackbar
The key successfully created. Add it to a Safe on desktop and then restart the mobile app.
```